src/katello/agent/katelloplugin.py

```diff
@@ -114,11 +114,10 @@ def setup_plugin():
     if not ConsumerIdentity.existsAndValid():
         # not registered
         return
-    cfg = plugin.cfg()
     rhsm_conf = Config(RHSM_CONFIG_PATH)
     certificate = ConsumerIdentity.read()
-    cfg.messaging.url = 'ssl://%s:5671' % rhsm_conf['server']['hostname']
-    cfg.messaging.uuid = 'pulp.agent.%s' % certificate.getConsumerId()
+    plugin.cfg.messaging.url = 'amqps://%s' % rhsm_conf['server']['hostname']
```

Comment:
This is just the changes needed for gofer 2.x (upgraded from 1.x) in general.  I have the changes needed to work with proton / dispatch router in a follow up PR.  Keeping them separate so the changes can be reviewed in isolation.

It would be better to get the port from the RHSM configuration file instead of hard coding but we'll need support in the file for that.  Not all consumers will be connecting to a dispatch router.